spec/build_spec.cr

```diff
@@ -35,7 +34,7 @@ module Amber::CLI
       puts "====== DONE Database created #{TEST_APP_NAME} ======"
 
       puts "RUNNING: shard update started..."
-      `shards update`
+      puts `shards update`
```

Comment:
Is puts here intentional? I thought we wanted to keep the specs logging to minimal